docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    working_dir: /cinema
+
+    command: >
+      sh -c "python manage.py migrate &&
+             python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
```

Comment:
The `depends_on` directive does not wait for the database to be ready; it only controls the startup order. Consider using a script or tool to wait for the database to be ready before starting the application, such as a `wait_for_db` command.